### PR TITLE
fix: missing profilingFunctionsConsumer.sidecars

### PR DESCRIPTION
**Pull request by ServerSideHannes**

### CODE REVIEW

File: charts/sentry/templates/snuba/deployment-snuba-profiling-profiles-consumer.yaml [hunk 1]
```diff
@@ -160,8 +160,8 @@ spec:
         securityContext:
 {{ toYaml .Values.snuba.profilingProfilesConsumer.containerSecurityContext | indent 12 }}
 {{- end }}
-{{- if .Values.profilingFunctionsConsumer.sidecars }}
-{{ toYaml .Values.profilingFunctionsConsumer.sidecars | indent 6 }}
+{{- if .Values.snuba.profilingFunctionsConsumer.sidecars }}
```
Comment (MemberIT) [inline charts/sentry/templates/snuba/deployment-snuba-profiling-profiles-consumer.yaml]: @ServerSideHannes please correct `profilingFunctionsConsumer` to `profilingProfilesConsumer` for this file

Comment (MemberIT) [reply charts/sentry/templates/snuba/deployment-snuba-profiling-profiles-consumer.yaml]: And please add default commented values `# sidecars: []` for profilingFunctionsConsumer and profilingFunctionsConsumer to `values.yaml`

**Comment by MemberIT** [comment]
@ServerSideHannes 
You still need to adjust the commit message, as "wip" is not informative, even though it passes the linter check. Please make the necessary changes.

**Comment by Mokto** [comment]
He doesn't need to, the title of the PR is good enough and I squash before merging.

**Review by Mokto — APPROVED** [review]

**Pull request merged**